### ISSUE TITLE
How do you access the results

### ISSUE THREAD

**Issue by marcuus** [fega/vue2-leaflet-geosearch#19]
The demo code seems to work fine putting the select result address into the input field.
But how does the parent component access this field?

**Comment by Guysnacho** [comment]
This was halfway answered in issue #3 but the way I handled it was like this 

```
mounted() {
    this.$refs.map.mapObject.on("geosearch/showlocation", response => {
      this.address = response.location.label;
    });
}
```

In the official [documentation](https://github.com/smeijer/leaflet-geosearch) it shows the format of results, if you handle it with "showlocation" then you get a payload with a both a location object and a marker object. The location object has everything you need and it follows the given format in the documentation. Hopefully that helps, I thought it was broken for a while but I

**Comment by darrenklein** [comment]
Yeah, it's referenced in the readme.md in the Node module, though it doesn't seem to come up on the Github landing page for this repo -


**Events**

`geosearch/showlocation` is fired when location is chosen from the result list.

```js
map.on('geosearch/showlocation', yourEventHandler);
```

`geosearch/marker/dragend` is fired when marker has been dragged.

```js
map.on('geosearch/marker/dragend', yourEventHandler);
```
